r-blacklist

```diff
@@ -0,0 +1,331 @@
+recipes/bioconductor-affyio
+recipes/bioconductor-annotate
+recipes/bioconductor-annotationdbi
+recipes/bioconductor-annotationforge
+recipes/bioconductor-basic4cseq
+recipes/bioconductor-biobase
+recipes/bioconductor-biocgenerics
+recipes/bioconductor-biocinstaller
+recipes/bioconductor-biocparallel
+recipes/bioconductor-biocstyle
+recipes/bioconductor-biomart
+recipes/bioconductor-biosigner
+recipes/bioconductor-biostrings
+recipes/bioconductor-biovizbase
+recipes/bioconductor-bsgenome
+recipes/bioconductor-bsgenome.celegans.ucsc.ce6
+recipes/bioconductor-bsgenome.dmelanogaster.ucsc.dm3
+recipes/bioconductor-bsgenome.ecoli.ncbi.20080805
+recipes/bioconductor-bsgenome.hsapiens.ucsc.hg19
+recipes/bioconductor-bsgenome.mmusculus.ucsc.mm9
+recipes/bioconductor-bubbletree
+recipes/bioconductor-bumphunter
+recipes/bioconductor-camera/1.22.0
+recipes/bioconductor-camera/1.26.0
+recipes/bioconductor-category
+recipes/bioconductor-cexor
+recipes/bioconductor-cghbase
+recipes/bioconductor-cghcall
+recipes/bioconductor-chipcomp
+recipes/bioconductor-chipseeker
+recipes/bioconductor-clusterprofiler
+recipes/bioconductor-cn.mops
+recipes/bioconductor-complexheatmap
+recipes/bioconductor-csaw
+recipes/bioconductor-dada2
+recipes/bioconductor-degreport
+recipes/bioconductor-deseq
+recipes/bioconductor-deseq2/1.10.1
+recipes/bioconductor-dexseq
+recipes/bioconductor-diffbind
+recipes/bioconductor-dnacopy
+recipes/bioconductor-do.db
+recipes/bioconductor-dose
+recipes/bioconductor-dupradar
+recipes/bioconductor-ebimage
+recipes/bioconductor-ebseq
+recipes/bioconductor-edger
+recipes/bioconductor-edger/3.12.1
+recipes/bioconductor-flowai
+recipes/bioconductor-flowcl
+recipes/bioconductor-flowcore
+recipes/bioconductor-flowdensity
+recipes/bioconductor-fourcseq
+recipes/bioconductor-gage
+recipes/bioconductor-gagedata
+recipes/bioconductor-genefilter
+recipes/bioconductor-genelendatabase/1.6.0
+recipes/bioconductor-geneplotter
+recipes/bioconductor-genomation
+recipes/bioconductor-genomeinfodb
+recipes/bioconductor-genomicalignments
+recipes/bioconductor-genomicfeatures
+recipes/bioconductor-genomicranges
+recipes/bioconductor-geoquery
+recipes/bioconductor-ggbio
+recipes/bioconductor-ggtree
+recipes/bioconductor-go.db
+recipes/bioconductor-gosemsim
+recipes/bioconductor-goseq/1.22.0
+recipes/bioconductor-gostats
+recipes/bioconductor-graph
+recipes/bioconductor-graphite
+recipes/bioconductor-gseabase
+recipes/bioconductor-gviz
+recipes/bioconductor-hpar
+recipes/bioconductor-illuminaio
+recipes/bioconductor-illuminamousev2.db
+recipes/bioconductor-impute
+recipes/bioconductor-iranges
+recipes/bioconductor-iranges/2.4.6
+recipes/bioconductor-iranges/2.4.7
+recipes/bioconductor-isomirs
+recipes/bioconductor-jmosaics
+recipes/bioconductor-kegg.db
+recipes/bioconductor-kegggraph
+recipes/bioconductor-keggrest
+recipes/bioconductor-limma
+recipes/bioconductor-limma/3.26.7
+recipes/bioconductor-limma/3.28.2
+recipes/bioconductor-limma/3.28.6
```

Comment:
I agree that older versions shouldn't be upgraded to R 3.3. But these infrastructure changes are generic, not just for R. For now I think the solution would be to keep those old versions on an "old version, do not migrate" blacklist.  The bioconda-utils tools make this easy to do, the only cost is in having another list to maintain -- which shouldn't be too bad since it won't change much.